File: CinemaReservations/Java/CinemaAllocationsAcceptanceTests/src/test/java/com/sdd/cinemaallocationsacceptancetests/helpers/stubmodel/SeatDto.java
package com.sdd.cinemaallocationsacceptancetests.helpers.stubmodel;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@Getter
@EqualsAndHashCode
public class SeatDto {

    @JsonProperty("Name")
    private String name;

    @JsonProperty("SeatAvailability")
    private String seatAvailability;
}

File: CinemaReservations/CSharp/CinemaAllocations.Tests.Acceptance/Helpers/SeatDto.cs
namespace CinemaAllocations.Tests.Acceptance.Helpers
{
    public class SeatDto
    {
        public SeatDto(string name, string seatAvailability)
        {
            Name = name;
            SeatAvailability = seatAvailability;
        }

        public string Name { get; }
        public string SeatAvailability { get; }
        
    }
}